lib/src/summarize.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dart_pad.summarize;
+
+class Summarizer {
```

Comment:
Let's add some brief dartdoc describing the class (or on the library declaration itself).